generators/server/index.js

```diff
@@ -558,4 +579,54 @@ module.exports = class JHipsterServerGenerator extends BaseBlueprintGenerator {
     if (useBlueprints) return;
     return this._end();
   }
+
+  _configureServer(config = this.jhipsterConfig) {
```

Comment:
Isn't it a derived thing? I would have let it in the gen-base derived props

Reply:
They are server configurations that depends on others configurations.
Otherwise we can have situations like https://github.com/jhipster/generator-jhipster/issues/16103.
Situations like this will not happen in the modular implementation.